### PR TITLE
Improve error message for tokenless authentication

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: launchable/commands/verify.py [hunk 1]
```diff
@@ -92,8 +92,13 @@ def verify(context: click.core.Context):
     try:
         res = client.request("get", "verification")
         if res.status_code == 401:
-            msg = ("Authentication failed. Most likely the value for the LAUNCHABLE_TOKEN "
-                   "environment variable is invalid.")
+            if os.getenv(TOKEN_KEY):
+                msg = ("Authentication failed. Most likely the value for the LAUNCHABLE_TOKEN "
+                       "environment variable is invalid.")
+            else:
+                msg = ("Authentication failed. Please set the LAUNCHABLE_TOKEN."
```
Comment (Konboi) [inline launchable/commands/verify.py:99]: Checked 

```sh
$ GITHUB_SHA=123abc GITHUB_EVENT_NAME='pull' GITHUB_ACTIONS=true GITHUB_RUN_ID=1 GITHUB_REPOSITORY=launchableinc/cli GITHUB_WORKFLOW=1 GITHUB_RUN_NUMBER=1 LAUNCHABLE_ORGANIZATION=launchableinc LAUNCHABLE_WORKSPACE=cli launchable verify
Organization: 'launchableinc'
Workspace: 'cli'
Proxy: None
Platform: 'macOS-14.1.1-arm64-arm-64bit'
Python version: '3.9.16'
Java command: 'java'
launchable version: '1.82.1.dev2+g7aa45d8.d20240206'
Authentication failed. Please set the LAUNCHABLE_TOKEN.If you intend to use tokenless authentication,kindly reach out to our support team for further assistance.
```

**Review by Konboi — APPROVED** [review]
LGTM

File: launchable/commands/verify.py (hunk 1 above)
Comment (Konboi) [reply launchable/commands/verify.py:99]: Ops need more space after `.`?

Comment (ono-max) [reply launchable/commands/verify.py:99]: Oh, that's right! Thanks!

Comment (ono-max) [reply launchable/commands/verify.py]: [546b553](https://github.com/launchableinc/cli/pull/758/commits/546b5530dc364ffb23b92de2eea514c682e85257)

**Review by Konboi — APPROVED** [review]

**Pull request merged**